From 4e0a8fc68bda2a88de5468a61218996a099d3302 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Wed, 12 Jun 2013 12:32:21 +0200
Subject: [PATCH 2/2] Revert "Revert "Fix fdo#64972 - strikethrough displays
 too high""

The underlying bug was fixed by the previous commit.

This reverts commit 1d34b905fad48b7aee7ccf39e33d952846f00fea.

Change-Id: I2333740ec623165b28253d20f0ba7fa2bcd19ac1
---
 vcl/generic/glyphs/gcach_ftyp.cxx | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/vcl/generic/glyphs/gcach_ftyp.cxx b/vcl/generic/glyphs/gcach_ftyp.cxx
index 162ba11..e34b38d 100644
--- a/vcl/generic/glyphs/gcach_ftyp.cxx
+++ b/vcl/generic/glyphs/gcach_ftyp.cxx
@@ -940,7 +940,6 @@ void ServerFont::FetchFontMetric( ImplFontMetricData& rTo, long& rFactor ) const
     rTo.mnDescent = 0;
     rTo.mnExtLeading = 0;
     rTo.mnSlant = 0;
-    rTo.mnIntLeading = (rTo.mnAscent + rTo.mnDescent) - ((maFaceFT->units_per_EM + 32) >> 6);
     rTo.mnWidth = mnWidth;
 
     // Calculating ascender and descender:
@@ -994,6 +993,8 @@ void ServerFont::FetchFontMetric( ImplFontMetricData& rTo, long& rFactor ) const
         }
     }
 
+    rTo.mnIntLeading = (rTo.mnAscent + rTo.mnDescent) - ((maFaceFT->units_per_EM + 32) >> 6);
+
     if( pOS2 && (pOS2->version != 0xFFFF) )
     {
         // map the panose info from the OS2 table to their VCL counterparts
-- 
1.8.2.1

